hooks/post_gen_project.py

```diff
@@ -2,79 +2,87 @@
 import json
 import shutil
 import subprocess
-from python_terraform import *
+
+from python_terraform import Terraform
 
 YES_OPTIONS, NO_OPTIONS = frozenset(["y"]), frozenset(["n", ""])
 PERM_OPTIONS = frozenset(["push", "pull", "admin"])
 
+
 def multiple_choices(question):
     selection = []
-    additional = "Would you like to add another (y/n)? "
+    additional = "Would you like to add another (y/n)? [n]: "
 
-    while True:
-        choice = raw_input(question).lower()
+    multi_q_check = True
+    while multi_q_check:
+        choice = input(question).lower()
         selection.append(choice)
-        if not check_choice(additional):
-            break
+        multi_q_check = check_choice(additional)
     return selection
 
+
 def check_choice(question):
-    while True:
-        choice = raw_input(question).lower()
-        if choice in YES_OPTIONS:
-            return True
-        if choice in NO_OPTIONS:
-            return False
+    choice = input(question).lower()
```

Comment:
This was a `while True` loop, and removing that puts us in a weird place in this function. It'll return `None` if the choice is neither in YES_OPTIONS nor NO_OPTION, and that `None` then needs to be handled in the caller.

Still, I think you're onto something. If we move the logic into the caller, we can make this whole function a single line

```
    return input(question).lower() in YES_OPTIONS
```

Reply:
since `None` is falsey I would expect that this single return should be enough and that additional work in the callers isn't necessary, correct?

Reply:
the difference from the while loop to no loop is more of a UI issue... previously, if you entered a yes value it returned and the caller handled it, if you entered a no value it returned and the caller handled _that_. if entered anything other than a yes or no value, it looped until you entered what it considered a valid value...

now, you _must_ enter a yes value for the return to be handled the same way. a no value _or any other non-yes value_ will all be falsey...